erdkotlin/src/main/java/com/elrond/erdkotlin/domain/transaction/models/Transaction.kt

```diff
@@ -11,6 +11,8 @@ data class Transaction(
     val sender: Address,
     val receiver: Address,
     val chainID: String,
+    val senderUsername: String = "",
```

Comment:
Do you think you can add an extra unit test for the transaction serialization / signing logic, a test that uses usernames as well?

Reply:
Done 👍